makei/crtfrmstmf.py

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-from makei.utils import format_datetime, objlib_to_path
-from makei.ibm_job import IBMJob, save_joblog_json
 import argparse
 from pathlib import Path
 import sys
 from typing import Optional
 from datetime import datetime
 sys.path.append(str(Path(__file__).resolve().parent.parent))
```

Comment:
So this adds ibmi-bob dir to the path?  or ?

Reply:
This will sets the PATH for the makei, crtfrmstmf, and getJobLog python process. So that they can find the `makei` package in that path